### PR TITLE
first part css

### PR DESCRIPTION
**Pull request by LetterBe**

### CODE REVIEW

File: frontend/src/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import {useNavigate} from "react-router-dom";
+import Button from "./css/Button";
+import React from "react";
+
+export default function NavBar() {
+    const navigate = useNavigate()
+
+    const routeToLoginAndRegiter = () => {
```
Comment (gossie) [inline frontend/src/NavBar.tsx:8]: Why do you need `routeToLoginAndRegiter` and `routeToLogin` when they both do the same.

Comment (gossie) [inline frontend/src/NavBar.tsx:8]: Why do you need `routeToLoginAndRegiter` and `routeToLogin` when they both do the same.

Comment (LetterBe) [reply frontend/src/NavBar.tsx:8]: I deleted it ! Actually, I dont

File: frontend/src/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import LogoOY from "./images/LogoOY.png"
+import React from "react";;
+export default function Header(){
+
+
+    return(
+        <div className='logoContainer'>
+            <img className='logo' src={LogoOY} alt=''/>
```
Comment (T-Kutzner) [inline frontend/src/Header.tsx:8]: better write something in alt, not just an empty string

File: frontend/src/todos/TodoPage.tsx [hunk 3]
```diff
@@ -27,16 +28,21 @@ export default function TodoPage() {
...
         <div className='app'>
-            <ol>
-                {todos.length > 0 && todos.map((todo) => <TodoItem key={todo.id} todoItem={todo}
-                                                                   onTodoSelected={setSelectedTodo}
-                                                                   onTodoChange={fetchAll}/>)}
-            </ol>
             <TodoEdit onTodoChange={fetchAll} todoToChange={selectedTodo}/>
-            <h6>Here you create new Tasks, edit them and,<br/> you you are done ckeck it,before deleting, <br/>so you get your score higher</h6>
+            <h6>Here you create new Tasks, edit them and,<br/> you you are done ckeck it,before deleting, <br/>so you
+                get your score higher</h6>
```
Comment (T-Kutzner) [inline frontend/src/todos/TodoPage.tsx:36]: you you
check statt ckeck

**Review by T-Kutzner — APPROVED** [review]
just two suggestions

**Pull request merged**